packages/components/src/Chips/ChipTypes.tsx

```diff
@@ -48,4 +49,16 @@ export interface InternalChipProps {
   onClick?(
     event: MouseEvent<HTMLDivElement | HTMLInputElement | HTMLButtonElement>,
   ): void;
+
+  /**
+   * Callback for keyboard interaction with chips (e.g., chip deletion).
+   */
+  onKeyDown?(
+    event: KeyboardEvent<XOR<HTMLDivElement, HTMLButtonElement>>,
+  ): void;
```

Comment:
I think XOR is technically more correct here?  Just because for it only makes sense to have an event of one or the other.  I am on the fence though, because it seems like with that `ts-xor` library you can only do two xors.  

And I think
```
XOR<XOR<HTMLDivElement, HTMLInputElement>, HTMLButtonElement>
```
Seems overly complicated, so I think just doing the `OR` makes sense.